app/src/main/java/org/simple/clinic/phone/PhoneNumberMaskerConfig.kt

```diff
@@ -6,7 +6,8 @@ import org.simple.clinic.remoteconfig.ConfigReader
 
 data class PhoneNumberMaskerConfig(
     val maskingEnabled: Boolean,
-    val proxyPhoneNumber: String
+    val proxyPhoneNumber: String,
+    val showPhoneMaskBottomSheet: Boolean = false
```

Comment:
Instead of using a default value in the constructor, can you set this in the `read` method in the companion instead?

Reply:
I have removed the `showPhoneMaskBottomSheet` property